packages/forma-36-react-components/src/components/TextInput/__snapshots__/TextInput.test.tsx.snap

```diff
@@ -2,11 +2,10 @@
 
 exports[`renders the component as required 1`] = `
 <div
-  className="TextInput TextInput--full"
+  className=""
```

Comment:
Looks like this is unexpected update and something is broken with styles import

Reply:
Good catch, didn't need to import styles as I initially thought, fixed